cve_bin_tool/helper_script.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
+import re
+import subprocess
+import sys
+
+from cve_bin_tool.cvedb import CVEDB, DBNAME, DISK_LOCATION_DEFAULT
+from cve_bin_tool.extractor import Extractor
+from cve_bin_tool.file import is_binary
+from cve_bin_tool.log import LOGGER
+from cve_bin_tool.strings import Strings
+from cve_bin_tool.util import DirWalk, inpath
+from cve_bin_tool.version_scanner import (  # using it just for clean_file_path
+    VersionScanner,
+)
+
+
+class HelperScript:
+    """Helps Contributors to understand how various common filenames, version-strings, etc. look"""
```

Comment:
This docstring could be a bit better:
- Contributor... to what?  of what?
- Why would these contributors want to know these things?
- What are they going to do with the output of the script?

I'd go with something more like 
"Helps contributors who want to write a new cve-bin-tool checker find common filenames, version strings, and other necessary data for building a binary checker."

Or if you'd like a one-liner, maybe just skip who's using it and explain only what it does.

"Helps find common strings and other components needed to build a cve-bin-tool checker"